templates/aws/cluster_nodes/manifest.yaml

```diff
@@ -0,0 +1,74 @@
+name: aws-cluster-nodes
+description: |
+  Creates ec2 instances in AWS with a load balancer, and a route 53 record entry.
+variables:
+  aws_access_key:
+    sensitive: true
+    type: string
+    optional: false
```

Comment:
I think `optional` should be false by default.

Reply:
Fixed